### DISCUSSION TITLE
Nova

### DISCUSSION THREAD

**Discussion by luangreydive** [deepgram/community#124] [General help]
Is nova being applied automatically if we are using the API?
What must we change to use it?
Does the pricing differs from the previous version?

Thank you!

**Comment by SandraRodgers** [comment]
Hi @luangreydive 

At this time, Nova is not being applied automatically if you are using our API. (This may change in the future, however.)

You can use Nova by adding `model=nova` to your request. For example: `https://api.deepgram.com/v1/listen?model=nova` Here is a Nova quickstart guide with code examples (includes examples with our SDKs): https://developers.deepgram.com/documentation/getting-started/nova-quickstart/

Nova is now our most affordable model (in addition to it being our most powerful model). You can compare the pricing for all plans/models by logging into your Deepgram account and then in the console, click on "Settings" in the left side nav, and then click the "Billing and Plans" tab. That gives a useful chart to compare all the pricing. But I definitely recommend using Nova since we have made it our most affordable option.

Sandra

**Comment by luangreydive** [comment]
Is it available in spanish?

Thank you!

El lun, 17 abr 2023 a las 13:22, Sandra Rodgers ***@***.***>)
escribió:

> Hi @luangreydive <https://github.com/luangreydive>
>
> At this time, Nova is not being applied automatically if you are using our
> API. (This may change in the future, however.)
>
> You can use Nova by adding model=nova to your request. For example:
> https://api.deepgram.com/v1/listen?model=nova Here is a Nova quickstart
> guide with code examples (includes examples with our SDKs):
> https://developers.deepgram.com/documentation/getting-started/nova-quickstart/
>
> Nova is now our most affordable model (in addition to it being our most
> powerful model). You can compare the pricing for all plans/models by
> logging into your Deepgram account and then in the console, click on
> "Settings" in the left side nav, and then click the "Billing and Plans"
> tab. That gives a useful chart to compare all the pricing. But I definitely
> recommend using Nova since we have made it our most affordable option.
>
> Sandra
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/deepgram/discussions/124#discussioncomment-5639292>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZYVRZ6B3LLJDCMTLAJ6IBLXBVU5BANCNFSM6AAAAAAW644FSI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Luan Rodriguez* *|* CEO

<https://www.greydive.com/?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
www.greydive.com
<https://www.greydive.com/?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
Lee los últimos artículos de nuestro blog
<https://www.greydive.com/blog?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
Suscríbete a e-commerce lab.
<https://www.greydive.com/e-commerce-lab?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>

**Reply by SandraRodgers** [reply]
You can see on [this page](https://developers.deepgram.com/documentation/features/language/#language-options) which languages are available with our different models.

For the time being, Nova is not available with Spanish. Our enhanced and base models are, and so is Deepgram Whisper Cloud (see [this page](https://developers.deepgram.com/documentation/guides/deepgram-whisper-cloud/#language-detection) for more info).

I hope this helps!

**Comment by luangreydive** [comment]
Thank you!
Please let me know once Nova is available in Spanish.

El lun, 17 abr 2023 a las 15:51, Sandra Rodgers ***@***.***>)
escribió:

> You can see on this page
> <https://developers.deepgram.com/documentation/features/language/#language-options>
> which languages are available with our different models.
>
> For the time being, Nova is not available with Spanish. Our enhanced and
> base models are, and so is Deepgram Whisper Cloud (see this page
> <https://developers.deepgram.com/documentation/guides/deepgram-whisper-cloud/#language-detection>
> for more info).
>
> I hope this helps!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/deepgram/discussions/124#discussioncomment-5640791>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZYVRZ6WE3JXTG5COLZDBELXBWGLVANCNFSM6AAAAAAW644FSI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Luan Rodriguez* *|* CEO

<https://www.greydive.com/?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
www.greydive.com
<https://www.greydive.com/?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
Lee los últimos artículos de nuestro blog
<https://www.greydive.com/blog?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
Suscríbete a e-commerce lab.
<https://www.greydive.com/e-commerce-lab?utm_source=firma-mail-luan&utm_medium=firma-mail-luan&utm_campaign=firma-mail-luan&utm_id=firma-mail-luan>
